content/en/blogs/kubesphere-certificates.md

```diff
@@ -0,0 +1,116 @@
+|Jian Zhang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/member-zhagnjian.png) |
+|Kai Zhang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/member-zhangkai.png) |
+
+## KubeSphere Talented Speaker
+
+KubeSphere Talented Speakers are awarded to outstanding lecturers who have conducted public technology presenation at KubeSphere Open-Source Live, KubeSphere Meetup 2020, and CNCF Webinar. They are excellent KubeSphere evangelists and splendid speakers.
+
+| Name | Certificate |
+| ---- | ---- |
+|Benjamin Huo|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-benjaminhuo.png) |
+|Tian Fang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-fangtian.png) |
+|Leo Liu|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-leoliu.png) |
+|Lei Song|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-songlei.png) |
+|Juan Xu|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-xujuan.png)|
+|Zhanling Chen|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-chenzhanling.png) |
+|Jeff Zhang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-jeffzhang.png) |
+|Yueliang Liu|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-liuyueliang.png) |
+|Tally Lee|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-tallylee.png) |
+|Xiaofei Yang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-yangxiaofei.png) |
+|Jiong Duan|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-duanjiong.png) |
+|Wanjun Lei|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-leiwanjun.png) |
+|Michael Yuan|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-michaelyuan.png) |
+|Wenhu Wang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-wangwenhu.png) |
+|Ke Zhou|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/certificates/speaker-zhouke.png) |
+
+## KubeSphere Ambassador
+
+KubeSphere Ambassador is awarded to technical evangelists who are helping grow the KubeSphere community. The following members have shared KubeSphere practical cases and technical articles in the KubeSphere community many times to help more users to get to know the application scenarios and best practices of KubeSphere.
```

Comment:
KubeSphere Ambassador is awarded to technical evangelists who are helping grow the KubeSphere community.
->
KubeSphere Ambassadors are awarded to technical evangelists who are helping to grow the KubeSphere community.